src/customisations/models/IMediaEventContent.ts

```diff
@@ -30,31 +34,67 @@ export interface IEncryptedFile {
     v: string;
 }
 
-export interface IMediaEventInfo {
-    thumbnail_url?: string; // eslint-disable-line camelcase
-    thumbnail_file?: IEncryptedFile; // eslint-disable-line camelcase
-    thumbnail_info?: {
-        // eslint-disable-line camelcase
-        mimetype: string;
-        w?: number;
-        h?: number;
-        size?: number;
-    };
-    mimetype: string;
+interface ThumbnailInfo {
+    mimetype?: string;
     w?: number;
     h?: number;
     size?: number;
 }
 
-export interface IMediaEventContent {
-    msgtype: string;
-    body?: string;
-    filename?: string; // `m.file` optional field
-    url?: string; // required on unencrypted media
-    file?: IEncryptedFile; // required for *encrypted* media
-    info?: IMediaEventInfo;
+interface BaseInfo {
+    mimetype?: string;
+    size?: number;
+}
+
+export interface FileInfo extends BaseInfo {
+    [BLURHASH_FIELD]?: string;
+    thumbnail_file?: IEncryptedFile;
+    thumbnail_info?: ThumbnailInfo;
+    thumbnail_url?: string;
+}
+
+export interface ImageInfo extends FileInfo, ThumbnailInfo {}
+
+export interface AudioInfo extends BaseInfo {
+    duration?: number;
```

Comment:
I think it would be good add explicitly in the field name or a comment that this is in milliseconds

Reply:
Sure will add some docs for these types